packages/standard-minifier-js/package.js

```diff
@@ -12,7 +12,11 @@ Package.registerBuildPlugin({
     'ecmascript'
   ],
   npmDependencies: {
+    'meteor-package-install-swc': '1.1.2',
+    'acorn': '8.10.0',
     "@babel/runtime": "7.18.9",
+    '@babel/parser': '7.22.7',
```

Comment:
Is @babel/parser still needed? Now that Acorn is available as a faster alternative, can we remove it?